config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -104,6 +104,7 @@ teams:
     - jsturtevant
     - mboersma
     - shysank
+    - dtzar
```

Comment:
the list is alphabetical order

Reply:
Also, wondering if we should just refresh this list in a single PR instead of having 4 different PRs open to add people (https://github.com/kubernetes/org/pulls?q=is%3Apr+is%3Aopen+cluster-api-azure-pms)... There are a few names in here that should be removed as well as they are no longer OWNERS. The list should be everyone in

```
  cluster-api-azure-maintainers:
    - CecileRobertMichon
    - devigned
    - jackfrancis
    - mboersma
  cluster-api-azure-reviewers:
    - Jont828
    - jsturtevant
    - marosset
    - nawazkh
    - nojnhuh
    - sonasingh46
    - willie-yao
```

+ yourself

Would you like to update in in this PR while you're here?

Reply:
Yes, I went ahead and did this.